.evergreen/config.yml

```diff
@@ -81,6 +81,10 @@ functions:
               export SSL=${SSL}
               export TOPOLOGY=${TOPOLOGY}
               export MONGODB_VERSION=${MONGODB_VERSION}
+
+              if [ "Windows_NT" != "$OS" ]; then
+                  ulimit -n 64000
```

Comment:
Is there any reason for 64000 specifically?

Reply:
This is the value evergreen hosts are supposed to have for max open files (and most do).